# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGClipPathFrame.h
# Commit: b16351df250c
# Full Hash: b16351df250c572a14efa126f7702fe104ac9fd7
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-07-22 21:26:48
# Regressor Bug: 1843945
# File Overlap Count: 1
# Description:
#   Bug 1843945 - optimise SVGObserverUtils r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183802
# ==============================================================================

diff -r 7e8063ebfd83 -r b16351df250c layout/svg/SVGClipPathFrame.h
--- a/layout/svg/SVGClipPathFrame.h	Sat Jul 22 10:03:30 2023 +0000
+++ b/layout/svg/SVGClipPathFrame.h	Sat Jul 22 14:06:38 2023 +0000
@@ -36,7 +36,8 @@
       : SVGContainerFrame(aStyle, aPresContext, kClassID),
         mIsBeingProcessed(false) {
     AddStateBits(NS_FRAME_IS_NONDISPLAY | NS_STATE_SVG_CLIPPATH_CHILD |
-                 NS_FRAME_MAY_BE_TRANSFORMED);
+                 NS_FRAME_MAY_BE_TRANSFORMED |
+                 NS_STATE_SVG_RENDERING_OBSERVER_CONTAINER);
   }
 
  public: